spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-fabric8-client-secrets-event-reload/src/test/resources/secret.yaml

```diff
@@ -6,4 +6,4 @@ metadata:
 data:
   # from.properties.key=initial
   application.properties: |
-    ZnJvbS5wcm9wZXJ0aWVzLmtleT1pbml0aWFsCg==
```

Comment:
`Cg==` is a space... yeah, took a bit of har pulling to understand where my test was failing.